tests/test_server/test_auth/test_keycloak_auth.py

```diff
@@ -25,14 +27,32 @@
     ],
     indirect=True,
 )
-async def test_get_keycloak_user_unauthorized(test_client: AsyncClient, mock_keycloak_well_known, caplog):
+async def test_get_keycloak_user_unauthorized(
+    test_client: AsyncClient,
+    mock_keycloak_well_known,
+    caplog,
+    server_app_settings: Settings,
+):
+    k_settings = KeycloakSettings.model_validate(server_app_settings.auth.keycloak)
```

Comment:
`k_settings` -> `keycloak_settings`

Reply:
change to `settings`. `keycloak_settings` - too long